visidata/loaders/postgres.py

```diff
@@ -115,17 +162,19 @@ def reload(self):
                 self.addRow(r)
 
     def openRow(self, row):
-        return PgTable(self.name+"."+row[0], source=row[0], sql=self.sql)
+        return PgTable(
+            self.name+"."+row[0],
+            source=(self.source or options.postgres_schema)+"."+row[0],
+            sql=self.sql
+        )
 
 
 # rowdef: tuple of values as returned by fetchone()
 class PgTable(Sheet):
     @asyncthread
     def reload(self):
-        if self.options.postgres_schema:
-            source = f"{self.options.postgres_schema}.{self.source}"
-        else:
-            source = self.source
+        source = self.source
```

Comment:
This will conflict with #2129, where the table name and schema were quoted to support both being something other than lowercase. 

i.e.:

`"public"."TableName"` rather than `public.TableName` or `"public.TableName"`

Reply:
Thanks for checking! As part of this PR, should we then move the #2129 fix into `openurl_postgres`?

Reply:
We will have to check that the loading of lowercase table names is preserved before merging.

Reply:
> should we then move the #2129 fix into `openurl_postgres`?

I don't see how that could be done. I think my tweak belongs in these places, where `PgTable.source` is set:

certainly here:
https://github.com/jrjsmrtn/visidata/blob/621dd88a75c6a133ce1a79cbd795c51fccd1d743/visidata/loaders/postgres.py#L167

maybe here?:
https://github.com/jrjsmrtn/visidata/blob/621dd88a75c6a133ce1a79cbd795c51fccd1d743/visidata/loaders/postgres.py#L123

I don't understand what some of the variables in this code refers to, such as `row[0]`, `columns[0:1]`, etc, so I'm on shaky ground here.

Reply:
`row[0]` is the `table_name` as per the `rowdef` comment at the top of the class.  `columns[0:1]` creates a list containing the first column (and an empty list if there are no columns).

I don't think the quoting should be done in the `schema.table` string given to the sheet; the SQL should do the quoting.  So this means that the schema and the table name should be passed separately to the postgres sheet.